db/re/item_db_usable.yml

```diff
@@ -50094,6 +50228,103 @@ Body:
       NoAuction: true
     Script: |
       percentheal 30,30;
+  - Id: 101279
+    AegisName: 2022_LunarNewYears_box
+    Name: New Year Gift Box
+    Type: Usable
+    Flags:
+      Container: true
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoAuction: true
+    Script: |
+      getgroupitem(IG_2022_LUNARNEWYEARS_BOX);
+  - Id: 101306
+    AegisName: Poenitentia_Hammer
+    Name: Poenitentia Refining Hammer
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* reformui 101306; */
+  - Id: 101307
+    AegisName: Barmund_Hammer
+    Name: Barmund Refining Hammer
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* reformui 101307; */
+  - Id: 101308
+    AegisName: True_GemShadow_Up
+    Name: True Gemstone Shadow Spellbook
+    Type: Usable
+    Weight: 10
+    Flags:
+      BuyingStore: true
+    Script: |
+      /* synthesisui 101308; */
```

Comment:
```suggestion
      laphine_synthesis();
```

Reply:
laphine_synthesis db entry is missing.

Reply:
It's just one more.. many items have the laphine_synthesis command without the db (`3Lv_9Refine_Weapon_7Gu`...) from the laphine PR